classes/cluster/mk22_stacklight_basic/fuel/config.yml

```diff
@@ -0,0 +1,52 @@
+classes:
+- service.git.client
+- system.linux.system.single
+- system.linux.system.repo.tcp_salt
+- system.openssh.client.lab
+- system.salt.master.git
+- system.reclass.storage.salt
+- system.salt.minion.pki.authority
+- system.salt.minion.pki.certificate
+- system.sphinx.server.doc.reclass
+- system.keystone.client.single
```

Comment:
move these to ctl01 please, that will be standard way to deliver client roles [1st member of each cluster]
```
  reclass:
    storage:
      node:
        openstack_control_node01:
          classes:
          - client.class
```